src/modules/transaction/components/TransactionForm.js

```diff
@@ -175,38 +175,34 @@ class TransactionForm extends Component {
             )}
             {error !== "" && <div class={styles.formError} id="error">{error}</div>}
             <fieldset disabled={formDisabled}>
-              <label>
-                {I18n.t("transaction.amount")}
-                <div class={styles.amountInput}>
-                  <input
-                    name="amount"
-                    type="number"
-                    id="inputAmount"
-                    min="1"
-                    max="999"
-                    className={styles.userSelection}
-                    value={amount}
-                    onInput={this.onInput}
-                    class={styles.amountInput}
-                  />
-                  <span class={styles.kudoCurrency}>₭</span>
-                </div>
-              </label>
-              <label>
-                {I18n.t("transaction.receiver")}
+              <div class={styles.amountInput}>
+                <input
+                  name="amount"
+                  type="number"
+                  id="inputAmount"
+                  min="1"
+                  max="999"
+                  className={styles.userSelection}
+                  value={amount}
+                  onInput={this.onInput}
+                  class={styles.amountInput}
+                />
+                <span class={styles.kudoCurrency}>₭</span>
+              </div>
 
-                {receiver.id !== "" ? (
-                  <SelectedUser
-                    user={receiver}
-                    clearSelection={this.clearSelection}
-                  />
-                ) : (
+              {receiver.id !== "" ? (
```

Comment:
What will happen if `receiver.id` is null (an empty string is not the same as `null`)?